### ISSUE TITLE
wo stack upgrade - php 7.2 notifies of php_admin_value[open_basedir] already exists

### ISSUE THREAD

**Issue by s-a-s-k-i-a** [WordOps/WordOps#159]
I just ran wo stack upgrade and got this message:

```
root@server:~# wo stack upgrade
Do you want to continue:[y/N]y
Updating APT packages           [OK]
Upgrading APT Packages          [OK]
Applying Nginx configuration templates
Testing Nginx configuration     [OK]
Restarting Nginx                [OK]
Configuring php7.2-fpm
While reading from '/etc/php/7.2/fpm/pool.d/www-two.conf' [line 24]: option 'php_admin_value[open_basedir]' in section 'www-two' already exists
```

Checked mentioned file and found two lines defining php_admin_value[open_basedir] at the end of the file:

```
php_admin_value[open_basedir] = "/var/www/:/usr/share/php/:/tmp/:/var/run/nginx-cache/"
php_admin_value[open_basedir] = "/var/www/:/usr/share/php/:/tmp/:/var/run/nginx-cache/:/dev/shm:/dev/urandom"
```

I removed both of them and re-ran wo stack upgrade and received the same outcome in terminal:

`While reading from '/etc/php/7.2/fpm/pool.d/www-two.conf' [line 24]: option 'php_admin_value[open_basedir]' in section 'www-two' already exists`

This is from wo log:

```
2019-09-24 15:36:41,669 (DEBUG) wo : [95mconfiguring php file /etc/php/7.2/fpm/php.ini[0m
2019-09-24 15:36:41,676 (DEBUG) wo : [95mWritting php configuration into /etc/php/7.2/fpm/php.ini[0m
2019-09-24 15:36:41,677 (DEBUG) wo : [95mwritting php7.2 configuration into /etc/php/7.2/fpm/php-fpm.conf[0m
2019-09-24 15:36:41,678 (DEBUG) wo : loading template file /usr/lib/wo/templates/php-fpm.mustache
2019-09-24 15:36:41,681 (DEBUG) wo : [95mWriting PHP 7.2 configuration into /etc/php/7.2/fpm/pool.d/www.conf[0m
2019-09-24 15:36:41,681 (DEBUG) wo : [95mCopying file, Source:/etc/php/7.2/fpm/pool.d/www.conf, Dest:/etc/php/7.2/fpm/pool.d/www-two.conf[0m
2019-09-24 15:36:41,682 (DEBUG) wo : [95mDoing search and replace, File:/etc/php/7.2/fpm/pool.d/www-two.conf,Source string:[www], Dest String:[www-two][0m
```

Any idea whats happening here?

**Comment by AdsonCicilioti** [comment]
My problem not is the same but i need disable open_basedir by custom directory in vhosts specifics.

Any help?

**Comment by VirtuBox** [comment]
Issue has been fixed with the latest WordOps release (v3.9.9.1).
@AdsonCicilioti, to disable open_basedir only in specifics vhosts, you will have to add custom php-fpm pools for those sites and to update your vhost.
